events/equeue/tests/prof.c

```diff
@@ -1,424 +0,0 @@
-/*
```

Comment:
This test file also should stay, as it's part of equeue library and still runs on CI (https://github.com/ARMmbed/mbed-os/blob/master/.travis.yml#L232).